openlabclient/README.md

```diff
@@ -0,0 +1,13 @@
+# OpenLab management tool
+
+This is the command line tool for OpenLab management.
+
+## How to use
+
+You can install from source or just use `pip install openlabclient`
+
+## Supported command
+
+### Labchecker
```

Comment:
Better to say:
## Supported feature
``openlab check``: Health check tool for checking OpenLab CI infrastructures.

Reply:
+1

Reply:
Don